crypto/evp_extra/p_dh_asn1.c

```diff
@@ -15,6 +15,7 @@
 
 #include "internal.h"
 #include "../internal.h"
+#include "../fipsmodule/cpucap/internal.h"
```

Comment:
Not sure why it's not needed in other files, I'm assuming the file is included via other headers because it has the capabilities checks. I didn't verify though.